Bundle/Command/MergeCategories.php

```diff
@@ -0,0 +1,169 @@
+<?php
+namespace KwcNewsletter\Bundle\Command;
+
+use KwcNewsletter\Bundle\Model\Newsletters;
+use KwcNewsletter\Bundle\Model\SubscribersToCategories;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Exception\RuntimeException;
+
+
+class MergeCategories extends Command
```

Comment:
du merged hier nicht die kategorien sonder die empfänger in den kategorien bitte umbennen in was besseres ;)